src/chat.rs

```diff
@@ -3283,6 +3360,86 @@ mod tests {
         assert_eq!(msg_text, draft_text);
     }
 
+    #[async_std::test]
+    async fn test_delete_draft() -> Result<()> {
+        let t = TestContext::new().await;
```

Comment:
It is generally better to create new Alice instead, so the test does not spend time generating the key. This is what two other tests below do.

Reply:
cool, i did not know that! was always thinking that `TestContext::new()` is maybe even more efficient - and often ran into the issue that that context is not "fake configured" and cannot send messages out :)

Reply:
Maybe I'm wrong though, the key is generated when you send the first message or run autoconfiguration, so it should not be triggered by creating chats and setting drafts locally.

The default test context does not have the address configured, so it is not so easy to send the message and trigger key generation.

For testing setting/deleting the draft having an unconfigured account is probably more efficient indeed, because you don't even load the preconfigured key.